src/engine-scripts/puppet/onReady.js

```diff
@@ -42,9 +40,16 @@ module.exports = async ( page, scenario ) => {
 		await require( './echo.js' )( page, hashtags );
 	}
 
+	if ( hashtags.includes( '#scroll' ) ) {
```

Comment:
#scroll needs to happen after pinnable elements and dropdowns are interacted with, or else those interactions will scroll to the top of the page again. #scroll also needs to happen before #search-focus because of the sticky header case